libsolidity/ast/Types.cpp

```diff
@@ -3010,9 +3010,9 @@ TypePointers FunctionType::returnParameterTypesWithoutDynamicTypes() const
 	return returnParameterTypes;
 }
 
-TypePointers FunctionType::parameterTypes() const
+TypePointers FunctionType::parameterTypes(bool _includeSelfParameter) const
```

Comment:
Could also write a new function for this if you think it's better (the parameterNames is actually not used).